eng/versioning/version_shared.py

```diff
@@ -34,7 +34,7 @@
 logging.getLogger().setLevel(logging.INFO)
 
 def path_excluded(path):
```

Comment:
What all does this exclude? I suspect it disables more then just the version increment.

Reply:
Nevermind I guess I should look at the script name. I only looked at the method name.